kebechet/managers/manager.py

```diff
@@ -63,7 +63,7 @@ def __init__(
 
         self.service = service
         self.project = service.get_project(namespace=self.owner, repo=self.repo_name)
-        self._repo = None
+        self._repo: git.Repo
```

Comment:
Shouldn't the change include `=` instead of `:` ?

Reply:
Rather than assignment this was supposed to be type defintion, which is why a colon is being used. Otherwise the pylance tests threw errors.

Reply:
I see, then can we assign the default value as well.
`self._repo: git.Repo = None`